src/app/utils/images/pokemon-images.ts

```diff
@@ -0,0 +1,5 @@
+import { pokemonImagesUrl } from '../url/images';
+
+export const getPokemonImageUrl = (id: number): string => {
```

Comment:
normally util function are static functions inside a class, so they are easier to test